ui/src/app/components/cromwell-configuration-panel.tsx

```diff
@@ -0,0 +1,204 @@
+import * as React from 'react';
+import { useEffect } from 'react';
+import * as fp from 'lodash/fp';
+
+import { AppType } from 'generated/fetch';
+
+import { Button } from 'app/components/buttons';
+import { FlexColumn, FlexRow } from 'app/components/flex';
+import { WarningMessage } from 'app/components/messages';
+import { styles } from 'app/components/runtime-configuration-panel/styles';
+import { Spinner } from 'app/components/spinners';
+import { appsApi } from 'app/services/swagger-fetch-clients';
+import {
+  withCdrVersions,
+  withCurrentWorkspace,
+  withUserProfile,
+} from 'app/utils';
+import { findMachineByName } from 'app/utils/machines';
+import { runtimeStore, useStore } from 'app/utils/stores';
+
+import { defaultCromwellConfig } from './apps-panel/utils';
+import { CostPredictor } from './cost-predictor';
+
+const { useState } = React;
+
+const cromwellSupportArticles = [
+  { text: 'How to run Cromwell in All of Us workbench?', link: '#' },
+  { text: 'Cromwell documentation', link: '#' },
+  { text: 'Workflow and WDL', link: '#' },
+  { text: 'Running and Autopause', link: '#' },
+  { text: 'Storage options', link: '#' },
+];
+
+const PanelMain = fp.flow(
+  withCdrVersions(),
+  withCurrentWorkspace(),
+  withUserProfile()
+)(
+  ({
+    analysisConfig,
+    workspace,
+    profileState,
+    creatorFreeCreditsRemaining,
+    onClose,
+  }) => {
+    const { profile } = profileState;
+
+    return (
+      <FlexColumn style={{ height: '100%' }}>
+        <div
+          data-test-id='runtime-create-panel'
+          style={{ ...styles.controlSection, marginTop: '1rem' }}
+        >
+          <CostPredictor
+            {...{
+              creatorFreeCreditsRemaining,
+              profile,
+              workspace,
+              analysisConfig,
+            }}
+            status={null}
+            onPause={Promise.resolve()}
+            onResume={Promise.resolve()}
+            appType={AppType.CROMWELL}
+          />
+          <WarningMessage>
+            This cost is only for running the Cromwell Engine, there will be
+            additional cost for interactions with the workflow.
+            <a href='#' style={{ marginLeft: '0.25rem' }}>
```

Comment:
These help links don't appear to go anywhere. Is that intentional?

Reply:
Yeah, that is intentional. I believe that the actual links are coming later.

Reply:
we should file a story to track that